src/app/components/vertical-tabs/index.html

```diff
@@ -20,28 +20,47 @@
   </sky-demo-page-properties>
 
   <sky-demo-page-properties sectionHeading="Vertical tab properties">
+    <sky-demo-page-property
+      propertyName="tabHeading"
+      defaultValue="false"
+    >
+    Specifies the tab header. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="active"
       isOptional="true"
     >
-    Indicates whether the tab is active when the tabset loads.
+    Indicates whether the tab is active when the tabset loads. This property accepts <stache-code>boolean</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="tabHeading"
+      propertyName="ariaControls"
+      isOptional="true"
     >
-    Specifies the tab header.
+    Specifies an ID that corresponds to the <stache-code>tabId</stache-code> and identifies the element that contains the content the vertical tab exposes. This property supports <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure">accessibility rules for disclosures</a>. This property accepts <stache-code>string</stache-code> values.
```

Comment:
I think this first sentence is hard to follow. I know I recommended calling out that this value corresponds to the value of the `tabId` property, but I think it would be clearer if we make that info a second sentence:

"Specifies an ID to identify the element that contains the content that the vertical tab exposes. This ID corresponds to the <stache-code>tabId</stache-code>."

What do you think?